en/docs/remote-apps/connect-remote-applications.md

```diff
@@ -0,0 +1,118 @@
+# Connect Remote Applications
+
+If you need to observe an external application that you have written in [Ballerina programming language](https://ballerina.io/), you can connect it to Choreo.
+
+## Prerequisites
+
+Create an application in Ballerina programming language. For instructions, see [Ballerina Documentation - Writing Your First Ballerina Program](https://ballerina.io/learn/user-guide/getting-started/writing-your-first-ballerina-program/).
+
+!!! Tip
+    In this guide, we are using the same Ballerina application that is created in the guide linked above.
+
+## Step 1: Generate and copy the Ballerina configurations
```

Comment:
Should we say "Generate and copy the Ballerina configuration"?

Reply:
Changed.